index/core.json

```diff
@@ -0,0 +1 @@
mpletion of the headers, it is possible to cause the HTTP server to abort from heap allocation failure. Attack potential is mitigated by the use of a load balancer or other proxy layer."},"58":{"cve":["CVE-2018-12122"],"vulnerable":"6 || 8 || 10 || 11","patched":"^6.15.0 || ^8.14.0 || ^10.14.0 || ^11.3.0","publish_date":"2018-11-27","author":"Matteo Collina","reported_by":"Jan Maybach","ref":"https://nodejs.org/en/blog/vulnerability/november-2018-security-releases/","type":"CWE-400: Uncontrolled Resource Consumption / Denial of Service","overview":"Slowloris HTTP Denial of Service: An attacker can cause a Denial of Service (DoS) by sending headers very slowly keeping HTTP or HTTPS connections and associated resources alive for a long period of time."},"59":{"cve":["CVE-2018-12123"],"vulnerable":"6 || 8 || 10 || 11","patched":"^6.15.0 || ^8.14.0 || ^10.14.0 || ^11.3.0","publish_date":"2018-11-27","author":"Matteo Collina","reported_by":"Martin Bajanik","ref":"https://nodejs.org/en/blog/vulnerability/november-2018-security-releases/","type":"CWE-115: Misinterpretation of Input","overview":"Hostname spoofing in URL parser for javascript protocol: If a Node.js application is using url.parse() to determine the URL hostname, that hostname can be spoofed by using a mixed case \"javascript:\" (e.g. \"javAscript:\") protocol (other protocols are not affected). If security decisions are made about the URL based on the hostname, they may be incorrect."},"60":{"cve":["CVE-2019-5737"],"vulnerable":"6 || 8 || 10 || 11","patched":"^6.17.0 || ^8.15.1 || ^10.15.2 || ^11.10.1","publish_date":"2019-02-28","author":"Matteo Collina","reported_by":"Marco Pracucci","ref":"https://nodejs.org/en/blog/vulnerability/february-2019-security-releases/","type":"CWE-400: Uncontrolled Resource Consumption / Denial of Service","overview":"An attacker can cause a Denial of Service (DoS) by establishing an HTTP or HTTPS connection in keep-alive mode and by sending headers very slowly thereby keeping the connection and associated resources alive for a long period of time. Attack potential is mitigated by the use of a load balancer or other proxy layer. This vulnerability is an extension of CVE-2018-12121, addressed in November and impacts all active release lines including 6, 8, 10 and 11."},"61":{"cve":["CVE-2019-5739"],"vulnerable":"6","patched":"^6.17.0","publish_date":"2019-02-28","author":"Matteo Collina","reported_by":"Timur Shemsedinov","ref":"https://nodejs.org/en/blog/vulnerability/february-2019-security-releases/","type":"CWE-400: Uncontrolled Resource Consumption / Denial of Service","overview":"Keep-alive HTTP and HTTPS connections can remain open and inactive for up to 2 minutes in Node.js 6.16.0 and earlier. Node.js 8.0.0 introduced a dedicated server.keepAliveTimeout which defaults to 5 seconds. The behavior in Node.js 6.16.0 and earlier is a potential Denial of Service (DoS) attack vector. Node.js 6.17.0 introduces server.keepAliveTimeout and the 5-second default."}}
```

Comment:
the diffs on this are going to be pretty unmanageable, and its hard to review. is it possible to format the index so its got a property or CVE per line? and to terminate it with a newline (to get rid of the red marker you can see above)?

Reply:
Made this happen with a one-line change 👍